src/javaharness/java/arcs/android/demo/ArcsService.java

```diff
@@ -0,0 +1,87 @@
+package arcs.android.demo;
+
+import android.app.Service;
+import android.content.Intent;
+import android.os.Handler;
+import android.os.IBinder;
+import android.os.Looper;
+import android.util.Log;
+import android.view.View;
+import android.webkit.JavascriptInterface;
+import android.webkit.WebResourceRequest;
+import android.webkit.WebResourceResponse;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+/**
+ * ArcsService wraps Arcs runtime. Other Android activities/services are expected to connect to
+ * ArcsService to communicate with Arcs.
+ */
+public class ArcsService extends Service {
+
+  private static final String TAG = "Arcs";
+
+  private static final String ARCS_API_NAME = "DeviceClient";
+  private WebView arcsWebView;
+
+  @Override
+  public void onCreate() {
+    super.onCreate();
+
+    Log.d(TAG, "onCreate()");
+
```

Comment:
TODO for future CL: I think we need to stick with flogger, but find a way to provide an android Log backend. I don't believe flogger-android is available. (https://github.com/google/flogger/issues/11)  (https://github.com/google/flogger/commit/3f82fdd9df5ef13243e32d43be42243b703f818d) But there might be a trivial hack with System Logger to forward to android Log.

Alternatively, we can inject out own Log interface with .d/.e/etc methods, and have the Web version use flogger, but the android version directly use Android Log.

Reply:
The latter one seems better to me as we can avoid a backend dependency.